py3status/modules/backlight.py

```diff
@@ -128,9 +143,6 @@ def post_config_hook(self):
             self._set_backlight_level(self.brightness_initial)
```

Comment:
the condition that leads to this line needs to be adjusted too I think, removing `self.command_available`